content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md

```diff
@@ -49,6 +49,7 @@ let you meet it.
 * To get the aggregator working in your environment, [configure the aggregation layer](/docs/tasks/extend-kubernetes/configure-aggregation-layer/).
 * Then, [setup an extension api-server](/docs/tasks/extend-kubernetes/setup-extension-api-server/) to work with the aggregation layer.
 * Read about [APIService](/docs/reference/kubernetes-api/cluster-resources/api-service-v1/) in the API reference
+*   Learn about [Declarative Validation Concepts]({{< relref "declarative-validation" >}}), an internal mechanism for defining validation rules that in the future will help support validation for extension API server development.
```

Comment:
I wouldn't use a `relref` here; just a site relative link like the others.

Reply:
Done, now uses site relative link:
```
*   Learn about [Declarative Validation Concepts](/docs/reference/using-api/declarative-validation.md)
```